VimeoNetworking/VimeoNetworking/Models/Spatial.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Spatial.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 1/6/17.
+//
+//
+//
+
+public class Spatial: VIMModelObject
+{
+    enum Projection: String
+    {
+        case Equirectangular = "equirectangular"
```

Comment:
I'm torn as to whether these strings should live here or not. I think maybe it's okay? I'm not sure putting them in a Constants struct would be much of an improvement so I guess just leave it.